PINCache/PINOperationQueue.m

```diff
@@ -136,6 +141,7 @@ - (void)cancelOperation:(id <PINOperationReference>)operationReference
       NSMutableOrderedSet *queue = [self operationQueueWithPriority:operation.priority];
       [queue removeObject:operation];
       [_queuedOperations removeObject:operation];
+      dispatch_group_leave(_group); // Add operation to queue
```

Comment:
Can you make the comment something like "Matching group_enter in addOperation:withPriority"